100-level-AsyncAPI-Intro/step2.md

```diff
@@ -0,0 +1,24 @@
+As Eve and Chan begin building a communication system using AsyncAPI inside Chan's spaceship control room, Chan asks Eve if there is a standardised way of defining the communication system or if she defines the communication systems differently based on the particular use case.
+
+Eve replies that AsyncAPI has a standardised way to define communication systems. 
+
+Eve tells Chan that the communication system is defined using an AsyncAPI document in a specific structure with certain elements or components on the root level. And that the asyncAPI document structure remains the same for different types of use cases. 
+
+As it is much easier to understand things in action, Eve explains each component of an asyncAPI document to Chan as they program each part.
+
+## Deep Dive into Various Components of AsyncAPI
+
+--> Mahfuza's content regarding various components of AsyncAPI document<--
+
+## Request/reply 
+As Eve and Chan almost finish setting up the communication system, Eve explains to Chan about Request/reply pattern, which is not a component but a common communication pattern. 
+
+Eve explains Chan that this is a type of communication pattern where there are two components, a requestor and a replier. 
```

Comment:
Just to avoid confusion with AsyncAPI components, I suggest we use a different word. Actor might work, but feel free to use a better one if you find it. 

```suggestion
Eve explains Chan that this is a type of communication pattern where there are two actors, a requestor and a replier. 
```